doc/user/content/integrations/cube.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Cube"
+description: "How to connect a Cube deployment to Materialize"
+aliases:
+  /third-party/cube/
+menu:
+  main:
+    parent: "integration-guides"
+    name: "Cube"
+---
+
+[Cube](https://www.cubejs.com/) is an open-source **Headless BI platform**. This integration guide will teach you how to connect Cube with Materialize in a few steps using the browser.
+
+## Pre-requisite
+
+Before following the steps, make sure to meet the next items:
+
+- Have a Cube account. Create [one here](https://cubecloud.dev/auth/signup).
+- A Materialize's region enabled.
```

Comment:
Is this needed? It's assumed that if someone landed in the Materialize Cube integration guide, they'll know they need Cube and Materialize! We could maybe make this a double whammy and add instructions also for self-managed Cube as a follow-up?

Reply:
Yeah. It is pretty obvious. At first, it was a mix of making it easier to sign up (CTA) and cloud awareness but later ended up without an impactful meaning.

I can also apply this point and the following one (consistent structure) to the **SSH Bastion** guide in another PR.